Storage/Storage/Model/ShippingLabelAddress+CoreDataProperties.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import CoreData
+
+
+extension ShippingLabelAddress {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<ShippingLabelAddress> {
+        return NSFetchRequest<ShippingLabelAddress>(entityName: "ShippingLabelAddress")
+    }
+
+    @NSManaged public var company: String
+    @NSManaged public var name: String
+    @NSManaged public var phone: String
+    @NSManaged public var country: String
+    @NSManaged public var state: String
+    @NSManaged public var address1: String
+    @NSManaged public var address2: String
+    @NSManaged public var city: String
+    @NSManaged public var postcode: String
```

Comment:
⚠️ This is missing `originShippingLabel` and `destinationShippingLabel`. I guess that's intentional, right? We probably don't need it.

Reply:
That must have been my oversight not regenerating the subclasses for `ShippingLabelAddress` when I made other changes, thanks for spotting this 🦅 👁️

Reply:
Added here: https://github.com/woocommerce/woocommerce-ios/pull/3168/commits/3cf9c86e0d00a6c19283ccc3c3c2492516d444bc